content/blog/2018-05-08-react-native-navigation.md

```diff
@@ -16,11 +16,17 @@ author:
 
 ### From Native to Cross-Platform
 
-Coming from a native iOS background the way React Navigation is set up seemed a little bizarre. I was so used to sub-classing Navigation Controllers, that scaffolding my app’s navigation stack via a constructor seemed a little odd (it still seems a little odd). It was also a little frustrating when the docs don’t really take you past the ‘Hello World’ set up. So, with this post, I aim to build a clear guide to setting up a `TabNavigator` based application with `StackNavigator`’s embedded in each tab. I’ll also cover some tips on UI customization and passing props between navigators.
```

Comment:
this opening read weird to me. I provided some thoughts for reworking.

I would avoid using phrases like `a little`.  Be more assertive with your claims.

Reply:
Also, with posts like this, always assume that your developers don't know anything about the stuff you are introducing. So if you are introducing `react-navigation` developers wont know what a `TabNavigator` is. Therefor, you should explain what it is right after you mention it.